Mathlib/Algebra/BigOperators/Associated.lean

```diff
@@ -19,7 +19,8 @@ and products of multisets, finsets, and finsupps.
 
 variable {α β γ δ : Type*}
 
--- the same local notation used in `Algebra.Associated`
+/-- Notation for two elements of a monoid are associated, i.e.
+if one of them is another one multiplied by a unit on the right.-/
```

Comment:
Grammer is wrong here. Maybe
```suggestion
/-- Notation for two elements of a monoid to be associated, i.e.
for one of them to be the other multiplied by a unit on the right. -/
```
or maybe just `@[inherit_doc]` as you already do in `Algebra.Polynomial.Splits`?